src/components/SelectCircle.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {StyleProp, View, ViewStyle} from 'react-native';
+import globalStyles from '@styles/styles';
+import themeColors from '@styles/themes/default';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+
+type SelectCircleProps = {
+    /** Should we show the checkmark inside the circle */
+    isChecked: boolean;
+
+    /** Additional styles to pass to SelectCircle */
+    styles?: StyleProp<ViewStyle>;
```

Comment:
Is there no way to define a default for this prop like []?

Reply:
Hey @cead22. Thanks for spotting this. It turns out that no action is needed, as RN+TS still work well when `styles` are undefined. Please see this:
https://github.com/Expensify/App/pull/29988/files#r1373207252
Especially the first example that passes no styles to `FullScreenLoadingIndicator`: https://github.com/Expensify/App/pull/29988/files#r1374544242